_layouts/call.html

```diff
@@ -0,0 +1,55 @@
+<!-- <div class="ui container divider"></div> -->
+
+<div id="{{ page.priority }}" class="ui {{ page.color | default: 'blue' }} grid aligned segment call">
+    <div class="one wide column"></div>
+
+    {% if page.tag == 'left' %}
+    <div class="six wide column">
+        <h1 class="ui header">
+          <i class="{{ page.icon }} icon"></i>{{ page.title }}
+        </h1>
+        <p>{{ page.description }}</p>
+        {% if page.tips %}
+        <div class="ui info message">
+            <div class="header"><i class="idea icon"></i> Bon à savoir</div>
+            <p>{{ page.tips }}</p>
+        </div>
+        {% endif %}
+    </div>
+    <div class="height wide column">
+        <div class="ui fluid action input">
+            <input id="curl{{ page.priority }}" type="text" value="curl {{ page.link }}" readonly>
+            <button class="ui right labeled icon button btn-cpy" data-clipboard-target="#curl{{ page.priority }}"><i class="copy icon"></i>Copier</button>
+        </div>
+        <div class="ui segment">
+            {{ content }}
+        </div>
+    </div>
+
+    {% else %} <!-- Same but reverse --->
+
+    <div class="height wide column">
+        <div class="ui fluid action input">
+            <input id="curl{{ page.priority }}" type="text" value="curl {{ page.link }}" readonly>
+            <button class="ui right labeled icon button btn-cpy" data-clipboard-target="#curl{{ page.priority }}"><i class="copy icon"></i>Copier</button>
```

Comment:
Keep the Semantic UI naming conventions, don't import Bootstrap's. You have a `right labeled icon button`, make it a `right labeled icon copy button` and target it with `.copy.button`, rather than creating a `btn-copy` class. The only point of Semantic UI is to make class names more readable.